### ISSUE TITLE
Container inspect not working

### ISSUE THREAD

**Issue by pavradev** [Khazrak/JDocker#3]
MacOS 10.11 with docker-engine 1.12.0 test fails:
```
    @Test
    public void testContainer() {
        try {
            ContainerCreationRequest containerCreationRequest = ContainerCreationRequest.builder()
                    .name("test-container")
                    .image("ekino/wiremock:2.1.11")
                    .build();
            dockerClient.createContainer(containerCreationRequest);
            dockerClient.start("test-container");
            DockerContainerInspect dockerContainerInspect = dockerClient.inspectContainer("test-container", false);
            assertThat(dockerContainerInspect, notNullValue());
        } finally {
            dockerClient.stop("test-container");
        }
    }
```

**Comment by Khazrak** [comment]
reproduced problem, will try to fix later today

**Comment by Khazrak** [comment]
Will build a new version with this fixed and some more tests for future security

**Comment by Khazrak** [comment]
@pavradev Published version 1.1.2 with fix. You can try it (github) or wait for maven to sync and test it.

PS: Also has a class EasyContainer that makes creating containers easier. (see integration tests for example of usage)

**Comment by aquaplanet** [comment]
MacOS 10.11.6

    ~/Codemint/Karl/JDocker(branch:master*) » ./gradlew test
    megaman@paperweight
    :compileJava UP-TO-DATE
    :processResources UP-TO-DATE
    :classes UP-TO-DATE
    :compileTestJava UP-TO-DATE
    :processTestResources UP-TO-DATE
    :testClasses UP-TO-DATE
    :test UP-TO-DATE

    BUILD SUCCESSFUL

    Total time: 1.087 secs
    ------------------------------------------------------------
    ~/Codemint/Karl/JDocker(branch:master*) » docker --version
    megaman@paperweight
    Docker version 1.12.5, build 7392c3b
    ------------------------------------------------------------
    ~/Codemint/Karl/JDocker(branch:master*) » java -version
    megaman@paperweight
    java version "1.8.0_91"
    Java(TM) SE Runtime Environment (build 1.8.0_91-b14)
    Java HotSpot(TM) 64-Bit Server VM (build 25.91-b14, mixed mode)
    ------------------------------------------------------------
    ~/Codemint/Karl/JDocker(branch:master*) » ./gradlew check
    megaman@paperweight
    :compileJava UP-TO-DATE
    :processResources UP-TO-DATE
    :classes UP-TO-DATE
    :compileTestJava UP-TO-DATE
    :processTestResources UP-TO-DATE
    :testClasses UP-TO-DATE
    :test UP-TO-DATE
    :check UP-TO-DATE

    BUILD SUCCESSFUL

    Total time: 1.016 secs

Tested latest commit right now so I guess you have fixed the issue!

**Comment by pavradev** [comment]
Still not working when container is started in custom network:
```
    @Test
    public void testContainer() {
        try {
            NetworkCreateRequest networkCreateRequest = NetworkCreateRequest.builder()
                    .driver("bridge")
                    .name("test-network")
                    .checkDuplicate(true)
                    .build();
            dockerClient.createNetwork(networkCreateRequest);

            ContainerCreationRequest containerCreationRequest = ContainerCreationRequest.builder()
                    .name("test-container")
                    .image("ekino/wiremock:2.1.11")
                    .hostConfig(HostConfig.builder()
                            .networkMode("test-network")
                            .build())
                    .build();
            dockerClient.createContainer(containerCreationRequest);
            dockerClient.start("test-container");
            DockerContainerInspect dockerContainerInspect = dockerClient.inspectContainer("test-container", false);
            assertThat(dockerContainerInspect, notNullValue());
        } finally {
            try {
                dockerClient.stop("test-container");
            } finally {
                try {
                    dockerClient.remove("test-container");
                } finally {
                    dockerClient.removeNetwork("test-network");
                }
            }
        }
    }
```

**Comment by Khazrak** [comment]
Found what the problem was, links and aliases was badly documentet, they where String but should have been List of Strings. Test is green after that fix. The fix will be in version 1.1.4 that I will release today.
